package.json

```diff
@@ -95,11 +96,21 @@
     ]
   },
   "babel": {
+    "presets": [
+      [
+        "@babel/preset-env",
+        {
+          "targets": {
+            "node": "12"
```

Comment:
this should target whatever is the lowest this library supports - node 8 from the looks of https://github.com/jest-community/jest-extended/blob/437d55ae2e5e3226c9c3cd8bc2317965f9a52a8f/.travis.yml#L3

Reply:
Is that still the case? I think that file hasn't been updated in a while. What does Jest support?

Reply:
Jest 26 and 27 supports node 10, but I'm fine with only supporting 12 and later here as 10 is EOL